src/main/java/com/buschmais/jqassistant/plugin/asciidocreport/SourceFileMatcher.java

```diff
@@ -64,7 +66,12 @@
                 if ("file".equals(url.getProtocol())) {
                     String path = url.getPath();
                     if (path.endsWith("/" + DEFAULT_INDEX_FILE)) {
-                        File file = new File(url.getFile());
+                        File file;
+                        try {
+                            file = new File(URLDecoder.decode(url.getFile(), "UTF-8"));
```

Comment:
There's a unit test for this class which should cover the URL decoding by checking a path containing spaces.

Reply:
Can you point me to that unittest? `SourceFileMatcherTest` does not contain such a test. As far as I can tell `AsciidocReportPluginTest.defaultIndexDocument()` is the only test that executes the affected code fragment.

Reply:
Good catch! It means that SourceFileMatcherTest#detectIndexFileFromRuleSources does not test what it is supposed to be covering...  
The reason is that the test method initializes the SourceFileMatcher with a rulesDirectory but it should be ```SourceFileMatcher sourceFileMatcher = new SourceFileMatcher(null, null, null);``` instead (maybe the API could need some  improvement here, but probably not in the scope of this PR).
Afterwards the test will fail as expected if the path contains white spaces. Proposing to change "src/test/resources/jqassistant/" to "src/test/resources/working dir/jqassistant/" for verifying this. Would it be Ok for you to include this in the PR?